src/multiselect.js

```diff
@@ -116,6 +116,23 @@ export class Multiselect {
     }
   }
 
+  /**
+   * Ignore multi-field updates within the given function.
+   * @param {function} func The function to call.
+   */
+  withoutMultiFieldUpdates(func) {
+    let oldGroup = Blockly.Events.getGroup();
+    // Note that this depends on the fact that eventListener_ will ignore events with a group ID.
+    if (!oldGroup) {
+      Blockly.Events.setGroup("ignoreMultiFieldUpdates");
```

Comment:
Is it necessary to use a fixed group ID here? If yes, I would suggest we make the group ID one of the method parameters and pass it here, to make it more flexible. If no, let's simply use `Blockly.Events.setGroup(true);`

Reply:
If it's not neccessary to have a fixed group ID, then let's still use the random one to avoid any possible issues caused by ID collision. 

```suggestion
      Blockly.Events.setGroup(true);
```

Reply:
Done.